doc/api/Daemon.md

```diff
@@ -0,0 +1,100 @@
+Daemon API
+===========
+
+This document contains detailed descriptions of the daemon's API routes. For an
+overview of the daemon's API routes, see [API.md#daemon](/doc/API.md#daemon).
+For an overview of all API routes, see [API.md](/doc/API.md)
+
+There may be functional API calls which are not documented. These are not
+guaranteed to be supported beyond the current release, and should not be used
+in production.
+
+Overview
+--------
+
+The daemon is responsible for starting and stopping the modules which make up
+the rest of Sia. It also provides endpoints for viewing build constants.
+
+Index
+-----
+
+| Route                                     | HTTP verb |
+| ----------------------------------------- | --------- |
+| [/daemon/constants](#daemonconstants-get) | GET       |
+| [/daemon/stop](#daemonstop-get)           | GET       |
+| [/daemon/version](#daemonversion-get)     | GET       |
+
+#### /daemon/constants [GET]
+
+returns the set of constants in use.
+
+###### JSON Response
+```javascript
+{
+  // Timestamp of the genesis block.
+  "genesistimestamp": 1433600000, // Unix time
+  // Maximum size, in bytes, of a block. Blocks larger than this will be
+  // rejected by peers.
+  "blocksizelimit": 2000000, // bytes
+  // Target for how frequently new blocks should be mined.
+  "blockfrequency": 600, // seconds per block
+  // Height of the window used to adjust the difficulty.
+  "targetwindow": 1000, // blocks
+  // Duration of the window used to adjust the difficulty.
+  "mediantimestampwindow": 11, // blocks
+  // How far in the future a block can be without being rejected.
+  "futurethreshold": 10800, // seconds
+  // TODO: how is this different from futurethreshold?
+  "extremefuturethreshold": 21600, // seconds
+  // Total number of siafunds.
+  "siafundcount": "10000",
+  // Fraction of each file contract payout given to siafund holders.
+  "siafundportion": "39/1000",
+  // Number of children a block must have before it is considered "mature."
+  "maturitydelay": 144, // blocks
+
+  // Number of coins given to the miner of the first block. Note that elsewhere
+  // in the API currency is typically returned in hastings and as a bignum.
+  // This is not the case here.
+  "initialcoinbase": 300000, // SiaCoins.
+  // Minimum number of coins paid out to the miner of a block (the coinbase
+  // decreases with each block). Note that elsewhere in the API currency is
+  // typically returned in hastings and as a bignum. This is not the case
+  // here.
+  "minimumcoinbase": 30000, // SiaCoins
+
+  // Initial target.
+  "roottarget": [0,0,0,0,32,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],
+  // Initial depth.
```

Comment:
These feel incomplete, though I'm not sure how you'd expand the documentation here without getting too technical. Maybe 'Target hash for the first block following the genesis block' and 'Initial amount of work assigned to the blockchain.'